### ISSUE TITLE
Next Reindeer timer appears and disappears constantly, and displays negative time

### ISSUE THREAD

**Issue by elstie** [Anahkiasen/cookie-monster#21]

**Comment by Anahkiasen** [comment]
Yes this is a known bug as there is no Reindeer anymore, I need to add a check. This made it into the release because I wanted to make an urgency release and fix the bugs later so people weren't stuck with a crashing plugin.

**Comment by elstie** [comment]
Ah. Thanks for taking the time to respond. I'll keep trying to find bugs if that's helpful to you. I'd try to help code it, but I have no knowledge of this kind of coding.
